<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTextEdit" name="textEditMain">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tabRead">
       <attribute name="title">
        <string>Отримати Дані</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonRead">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Отримати Дані!</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>21</x>
          <y>10</y>
          <width>379</width>
          <height>48</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="labelAddr">
           <property name="text">
            <string>Адреса пристрою</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxDeviceAddr">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Кількість записів</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBoxRecordNum">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRegType">
           <property name="text">
            <string>Тип комірки</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxType"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelAddressRegister">
           <property name="text">
            <string>Адреса комірки</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEditRegAddr"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWrite">
       <attribute name="title">
        <string>Записати Дані</string>
       </attribute>
       <widget class="QPushButton" name="pushButtonWrite">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Записати Дані</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>11</y>
          <width>415</width>
          <height>74</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="labelAddrWrite">
           <property name="text">
            <string>Адреса пристрою</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="spinBoxDeviceAddrWrite">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>16</number>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Кількість записів</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSpinBox" name="spinBoxRecordNum_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRegTypeWrite">
           <property name="text">
            <string>Тип комірки</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxTypeWrite"/>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="labelAddressRegisterWrite">
           <property name="text">
            <string>Адреса комірки</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="lineEditRegAddrWrite"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelContentWrite">
           <property name="text">
            <string>Зміст</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditData">
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCOM">
    <property name="title">
     <string>COM-порт ПК</string>
    </property>
    <addaction name="actionConnect"/>
   </widget>
   <addaction name="menuCOM"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Параметри</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>openComPortDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openComPortDialog()</slot>
 </slots>
</ui>
